sites/hpnonline.com/server/templates/website-section/continuing-education.marko

```diff
@@ -40,6 +40,24 @@ $ const description = (data.descrtiption) ? data.description : "";
             </default-theme-breadcrumbs-with-home>
             <h1 class="page-wrapper__title">${title}</h1>
             <div class="page-wrapper__deck">${description}</div>
+            <if(alias === "continuing-education")>
```

Comment:
This should likely be setup as an ad unit and trafficked through GAM. @brandonbk thoughts?

Reply:
Ah, similar to the TI page? https://www.electronicdesign.com/design-learning-center

Reply:
@zarathustra323 - agreed, or it should leverage the newly added "logo" field on sections.  The issue with the latter is having to deal with multiple logos (unless they create a combined one).

Either way it should be set up to be maintainable long-term outside of our code.  Ads are the most logical choice, possibly followed by GTM (although that could become a rabbit hole).

Reply:
I reached out to Debra in regards to making these sponsorship links ads. Let me know if you want me to proceed with setting something up similar to the page I linked above, or if you have a different preference.

Reply:
@brandonbk I found something while consolidating the repo... sponsored company logos are already displaying on _most_ of the legacy Cygnus sites. They use the "Sponsored" scheduling option. Since this is already live on several sites, should this simply be added to HPN?  

Here's an example from `aviationpros.com/server/templates/website-section.index.marko` line 45...

```marko
<div class="row">
  <div class="col-lg-12 mb-block">
    <div class="node-list node-list--inner-justified sponsored">
      <marko-web-query|{ nodes }|
        name="website-scheduled-content"
        params={
          sectionId: id,
          optionName: "Sponsored",
          limit: 4,
          includeContentTypes: ["Company"],
          queryFragment
        }
      >
        <div class="node-list__header">Featured Companies</div>
        <website-content-card-deck-flow nodes=nodes cols=4 display-ads=false with-teaser=false />
      </marko-web-query>
    </div>
  </div>
</div>
```

Reply:
Consolidating the repo?

Reply:
@Alixers505 - hold off on this one for the time being; probably best to close the PR for now.  I met with Deb and Monique; the full request is actually more than just adding logos so they are going back to the group to get the full picture.  Once they have that, we'll re-assess.

Consolidating the repo = the rollup I mentioned on our call earlier this week to structure it more like the Informa & ACBM repos.

@zarathustra323 - that block is effectively Buyer's Guide Featured Companies.  In the interest of standardization, it can be included in your rollup; it will just end up collapsing for most groups.  To your point about it _basically_ being the same thing as the above request, it is a potential solution I can suggest to Monique/Deb depending on where this request ends up.